recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - toml
   run:
     - python
-    - pywin32-ctypes  # [win]
+    - pywin32-ctypes !=0.1.0,!=0.1.1  # [win]
```

Comment:
We may as well make the requirement `>0.1.1` rather than make these oddly-specific exclusions.